helm/prometheus-rules/templates/platform/atlas/alerting-rules/storage.rules.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+{{- if not .Values.mimir.enabled }}
+    cluster_type: "management_cluster"
+{{- end }}
+  name: observability.storage.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: observability.storage
+    rules:
+    - alert: ObservabilityStorageSpaceTooLow
```

Comment:
What is the reason for renaming from `DataDiskPersistentVolumeSpaceTooLow` and moving this alert from `storage.management-cluster.rules.yml` ?

Reply:
The renaming is not mandatory but the role was moved because storage.management-cluster.rules.yml belongs to turtle while this one belongs to atlas

Reply:
I can rename the alert if you prefer :)